sys/src/linux/loongarch64.rs

```diff
@@ -0,0 +1,2413 @@
+pub const LV2_UNITS__coef: &[u8; 43usize] = b"http://lv2plug.in/ns/extensions/units#coef\0";
+pub const LV2_UNITS__conversion: &[u8; 49usize] =
+    b"http://lv2plug.in/ns/extensions/units#conversion\0";
+pub const LV2_UNITS__db: &[u8; 41usize] = b"http://lv2plug.in/ns/extensions/units#db\0";
+pub const LV2_UNITS__degree: &[u8; 45usize] = b"http://lv2plug.in/ns/extensions/units#degree\0";
+pub const LV2_UNITS__frame: &[u8; 44usize] = b"http://lv2plug.in/ns/extensions/units#frame\0";
+pub const LV2_UNITS__hz: &[u8; 41usize] = b"http://lv2plug.in/ns/extensions/units#hz\0";
+pub const LV2_UNITS__inch: &[u8; 43usize] = b"http://lv2plug.in/ns/extensions/units#inch\0";
+pub const LV2_UNITS__khz: &[u8; 42usize] = b"http://lv2plug.in/ns/extensions/units#khz\0";
+pub const LV2_UNITS__km: &[u8; 41usize] = b"http://lv2plug.in/ns/extensions/units#km\0";
+pub const LV2_UNITS__m: &[u8; 40usize] = b"http://lv2plug.in/ns/extensions/units#m\0";
+pub const LV2_UNITS__mhz: &[u8; 42usize] = b"http://lv2plug.in/ns/extensions/units#mhz\0";
+pub const LV2_UNITS__midiNote: &[u8; 47usize] = b"http://lv2plug.in/ns/extensions/units#midiNote\0";
+pub const LV2_UNITS__mile: &[u8; 43usize] = b"http://lv2plug.in/ns/extensions/units#mile\0";
+pub const LV2_UNITS__min: &[u8; 42usize] = b"http://lv2plug.in/ns/extensions/units#min\0";
+pub const LV2_UNITS__mm: &[u8; 41usize] = b"http://lv2plug.in/ns/extensions/units#mm\0";
+pub const LV2_UNITS__ms: &[u8; 41usize] = b"http://lv2plug.in/ns/extensions/units#ms\0";
+pub const LV2_UNITS__name: &[u8; 43usize] = b"http://lv2plug.in/ns/extensions/units#name\0";
+pub const LV2_UNITS__oct: &[u8; 42usize] = b"http://lv2plug.in/ns/extensions/units#oct\0";
+pub const LV2_UNITS__pc: &[u8; 41usize] = b"http://lv2plug.in/ns/extensions/units#pc\0";
+pub const LV2_UNITS__prefixConversion: &[u8; 55usize] =
+    b"http://lv2plug.in/ns/extensions/units#prefixConversion\0";
+pub const LV2_UNITS__render: &[u8; 45usize] = b"http://lv2plug.in/ns/extensions/units#render\0";
+pub const LV2_UNITS__s: &[u8; 40usize] = b"http://lv2plug.in/ns/extensions/units#s\0";
+pub const LV2_UNITS__semitone12TET: &[u8; 52usize] =
+    b"http://lv2plug.in/ns/extensions/units#semitone12TET\0";
+pub const LV2_UNITS__symbol: &[u8; 45usize] = b"http://lv2plug.in/ns/extensions/units#symbol\0";
+pub const LV2_UNITS__unit: &[u8; 43usize] = b"http://lv2plug.in/ns/extensions/units#unit\0";
+pub const LV2_URI_MAP_URI: &[u8; 33usize] = b"http://lv2plug.in/ns/ext/uri-map\0";
+pub const LV2_URI_MAP_PREFIX: &[u8; 34usize] = b"http://lv2plug.in/ns/ext/uri-map#\0";
+pub const LV2_WORKER_URI: &[u8; 32usize] = b"http://lv2plug.in/ns/ext/worker\0";
+pub const LV2_WORKER_PREFIX: &[u8; 33usize] = b"http://lv2plug.in/ns/ext/worker#\0";
+pub const LV2_WORKER__interface: &[u8; 42usize] = b"http://lv2plug.in/ns/ext/worker#interface\0";
+pub const LV2_WORKER__schedule: &[u8; 41usize] = b"http://lv2plug.in/ns/ext/worker#schedule\0";
+#[doc = " The header of an atom:Atom."]
+#[repr(C)]
```

Comment:
This must be 64-bit-laligned, see: https://github.com/RustAudio/rust-lv2/commit/44aac000be49216ddd00f46aa7e8a05bde4a09f7#diff-20819adfed11ea1eee2eee94f933e4171c6181253a631cae5b3af100518e3267R688

Reply:
Thanks, I have changed.

Reply:
Please squash these two commits. There is no need to have two commits.

Reply:
Done.